index.js

```diff
@@ -13,6 +14,14 @@ class GEmojiElement extends HTMLElement {
     }
   }
 
+  get tone(): number {
+    return this.hasAttribute('tone') ? parseInt(this.getAttribute('tone'), 10) : 0
```

Comment:
Considering this as an enumerated attribute, I think the getter should always return valid values. Similar to how [the `autocomplete` attribute behaves](https://html.spec.whatwg.org/#dom-fe-autocomplete): 

> The autocomplete IDL attribute, on getting, must return the element's IDL-exposed autofill value, and on setting, must reflect the content attribute of the same name.

```suggestion
    const tone = parseInt(this.getAttribute('tone'), 10)
    return MODIFIERS.includes(tone) ? tone : 0
```